### DISCUSSION TITLE
Troubleshooting the 'Didn't Find LE Internal Object' Error in Version 1.8.0

### DISCUSSION THREAD

**Discussion by dz3n** [rpldy/react-uploady#677] [Q&A]
After upgrading to version 1.8.0, an error message, "Didn't find LE internal object. Something very bad happened!", has started to appear in my logs. Does anyone have more information about this issue?

**Comment by yoavniran** [comment]
hey @dz3n 
this is surprising since: a) I dont think this issue was ever reported. b) there wasnt any recent change in that area.

Please make sure you're using the same version (1.8.0) for all @rpldy/* dependencies and if you can, share more info about your implementation or better yet, a reproducing code/sandbox

a stacktrace of the error would also be useful.

**Reply by dz3n** [reply]
In my testing environment, the issue doesn't seem to occur, making it difficult to reproduce. However, I've recently encountered this error on one of my production servers, prompting me to investigate its cause.  The stacktrace is "compiled", so the information there is not that useful. The user agent associated with the error is: **Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36.**

If I come across any additional information, I'll be sure to share it.
